website/internal/views/devices.py

```diff
@@ -2,7 +2,7 @@
 
 import data_interface
 from internal import internal_site
-from internal.views.forms import AddNewDeviceForm, SetThermostatTargetForm
+from website.forms import AddNewDeviceForm, SetThermostatTargetForm
```

Comment:
There is no module named website. The root directory of the website component is already that directory. If you want to keep the forms.py file there, you should import it as:
from forms import ...